src/Tomcat/TomcatController.ts

```diff
@@ -60,43 +60,29 @@ export class TomcatController {
             throw (new Error(DialogMessage.noServer));
         }
 
-        const exist: boolean = await Utility.openFileIfExists(tomcatServer.getServerConfigPath());
-        if (!exist) {
-            throw (new Error(DialogMessage.noServerConfig));
+        const configFile: string = tomcatServer.getServerConfigPath();
+        if (!fse.pathExistsSync(configFile)) {
```

Comment:
better not use the sync version of this method

Reply:
Besides why remove the utility function openifexists? The method could be used later if we allow user to open/edit other configuration files

Reply:
Open/edit configuration files or any files cost only two blocks of logic: check if exists, then open it. Those two are covered in fes and vscode packages.

Reply:
Without a method, every time we want to open a file, we need type these logic.  Extract a method is more convenient.